static/app/views/dashboards/widgets/timeSeriesWidget/plottables/continuousTimeSeries.spec.tsx

```diff
@@ -0,0 +1,71 @@
+import {TimeSeriesFixture} from 'sentry-fixture/timeSeries';
+
+import LineSeries from 'sentry/components/charts/series/lineSeries';
+
+import {ContinuousTimeSeries} from './continuousTimeSeries';
+import type {Plottable} from './plottable';
+
+describe('ContinuousTimeSeries', () => {
+  describe('isEmpty', () => {
+    it('plottables as not empty', () => {
```

Comment:
big nit: test cases are best read as sentences starting with 'it', so '[it] considers plottables as not empty' or something like that would be better

Reply:
Oh, I accidentally a word. It should start with "Marks", thanks!